aggregator/apps/api/v1/stitcher.py

```diff
@@ -2,6 +2,43 @@
 from django.urls import reverse
 
 
+def ballot_charisma(ballot, sort_keys):
+    charisma_map = {
+        "ref": {"default": 100},
+        "parl": {"default": 90},
+        "europarl": {"default": 80},
+        "mayor": {"default": 70, "local-authority": 65},
+        "naw": {"default": 60},
+        "sp": {"default": 60},
+        "nia": {"default": 60},
+        "gla": {"default": 60},
+        "pcc": {"default": 50},
+        "local": {"default": 40},
+    }
+    modifier = 0
+    ballot_paper_id = ballot["ballot_paper_id"]
+    values_for_type = charisma_map[ballot_paper_id.split(".")[0]]
+    base_charisma = values_for_type.get(
+        sort_keys.get(ballot_paper_id), values_for_type.get("default")
+    )
```

Comment:
I found the way this line is written a bit non-obvious to read.
Do you think

```py
sort_key = sort_keys.get(ballot_paper_id, "default")
base_charisma = values_for_type.get(sort_key)
```

would be slightly clearer?

Reply:
I've had a go at making the logic clearer. Your example doesn't work, as you need to look up the organisation type and then the default (so, you need two `get()` calls).

See what you think, and if it's still not clear I'll have a rethink